src/fbw/src/failures/FailuresConsumer.cpp

```diff
@@ -50,3 +50,12 @@ bool FailuresConsumer::setIfFound(double identifier, bool value) {
     return true;
   }
 }
+
+bool FailuresConsumer::isAnyActive() {
+  if (std::any_of(activeFailures.begin(), activeFailures.end(), [](auto pair) {
```

Comment:
Vscode tells me that `std::any_of` needs to have `algorithm` included? Not sure why it compiles without it

Reply:
My IDE shows that it seems to be included down the line somewhere - but maybe better to add it explicitly.